### DISCUSSION TITLE
Possible to "untrack" an installed cask?

### DISCUSSION THREAD

**Discussion by millerlarson** [Homebrew/discussions#2936] [Casks] [answered]
Looking to attempt to stop double upgrading vscode.

Currently have virtual-studio-code installed via brew, but vscode often self-updates before brew gets to it.

Is there way to stop brew from managing this cask without uninstalling the cask?  I'd like to just tell brew to leave it alone, but from what I understand, `pin` doesn't work for casks...

**Comment by SMillerDev** [comment]
It won't auto update unless you force it with `--greedy`

**Reply by millerlarson** [reply]
Right, but I have other casks I _do_ want to keep updated with the use of `--greedy`

It just gets more complicated and I didn't know if I could install something with brew, and then tell brew "hands off" without having to remove/reinstall the app...

Moreover, it doesn't quite answer the question:  Is there a way to get brew to stop tracking a cask _without_ uninstallation/removal?

**Reply by SMillerDev** [reply]
The answer to the question is no, either brew manages it or it's managed by the user. There is no transitioning between the two.

**Reply by millerlarson** [reply]
Thank you, I understood. I'll either work around it, or just install it manually and avoid the issue.

**Comment by neutric** [comment, marked as answer]
```
brew upgrade --cask $(brew outdated --cask --greedy | grep -Eiv "visual-studio-code|google-chrome")
```
might work for you. The part in `$()` compiles a list of updatable casks where certain casks are filtered out by `grep`. I added `google-chrome` as second example item so that you see how to specify multiple items. This list is then handed to `brew upgrade`.

**Comment by artyomxx** [comment]
Stumbled upon this too, found the answer [on Stackoverflow](https://stackoverflow.com/questions/53301620/how-can-i-remove-a-cask-without-uninstalling-the-application):
> Delete the folder of the app you want to forget from /usr/local/Caskroom. On Apple silicon Macs, the location is /opt/homebrew/Caskroom
